### PR TITLE
[Issue20]: Add in WTFPL

### PR DESCRIPTION
**Pull request by cht8687**
#20 Add in WTFPL licence.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,3 +1,8 @@
+/* This program is free software. It comes without any warranty, to
+     * the extent permitted by applicable law. You can redistribute it
```
Comment (stevemao) [inline index.js:2]: Doesn't look like it's aligned properly. Is it intentional?

**Review by stevemao — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (cht8687) [reply index.js:2]: Directly copy and paste from official doc.

Comment (cht8687) [reply index.js:2]: @stevemao  http://www.wtfpl.net/faq/

Comment (cht8687) [reply index.js:2]: shall we align it?

**Pull request merged**